bottles/frontend/ui/details-bottle.blp

```diff
@@ -148,78 +115,151 @@ template DetailsBottle : .AdwPreferencesPage {
   }
   .AdwPreferencesGroup {
     Box {
+      orientation: vertical;
+
+      Label label_name {
+        halign: center;
+        label: _("My bottle");
+
+        styles [
+          "title-1",
+        ]
+      }
+
       Box {
-        orientation: vertical;
+        spacing: 6;
+        halign: center;
 
-        Label label_name {
-          halign: start;
-          label: _("My bottle");
+        Label label_arch {
+          label: _("Win64");
 
           styles [
-            "title-2",
+            "heading",
           ]
         }
 
-        Box {
-          spacing: 6;
+        Label {
+          label: "·";
```

Comment:
The separator characters are different now :\

Reply:
What's different?